patches/chrome-browser-resources-settings-privacy_page-personalization_options.ts.patch

```diff
@@ -16,7 +16,7 @@ index e6deebea7f7098e2e2a2233de789e6ec616ff3d6..500d6eb5c0a504b5407481598dba816f
    syncStatus: SyncStatus;
  
 -  // <if expr="_google_chrome and not chromeos_ash">
-+  // <if expr="_chromium and not chromeos_ash">
++  // <if expr="True and not chromeos_ash">
```

Comment:
can this just be `not chromeos_ash`?

Reply:
Fixed in a3821757f9dcdd6716e885aca327b6bba3467938.